changelog.txt

```diff
@@ -11,6 +11,7 @@
 * Add	- Allows registration of additional accounts.
 * Tweak - Carrier description on dynamic carrier registration form.
 * Fix   - Adjust documentation links.
+* Tweak - Adds WCCom access token and site ID to connect server request headers.
```

Comment:
Changelog update should go up in the 1.25.5 section

Reply:
Whoops, branch needed a belated update and I missed this one--just corrected the changelog entry now, cheers!